### PR TITLE
show notification on copy to clipboard

### PR DESCRIPTION
**Pull request by dkent600**
Resolves: https://github.com/daostack/alchemy/issues/1715

### CODE REVIEW

File: src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx [hunk 1]
```diff
@@ -33,8 +44,19 @@ export default class ProposalSummary extends React.Component<IProps, IState> {
     this.setState({ network: (await getNetworkName()).toLowerCase() });
   }
 
-  private copySchemeAddressOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void => copyToClipboard(schemeRegistrar.schemeToRegister);
-  private copySchemeParamsHashOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void => copyToClipboard(schemeRegistrar.schemeToRegisterParamsHash);
+  private copySchemeAddressOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void =>
+  {
+    const { showNotification } = this.props;
+    copyToClipboard(schemeRegistrar.schemeToRegister);
```
Comment (orenyodfat) [inline src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx:50]: can you make it one function which get the object to copy as a param ?schemeRegistrar.schemeToRegister or schemeRegistrar.schemeToRegisterParamsHas

File: src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx [hunk 2]
```diff
@@ -33,8 +44,19 @@ export default class ProposalSummary extends React.Component<IProps, IState> {
...
+    showNotification(NotificationStatus.Success, "Copied to clipboard!");
+  }
+
+  private copySchemeParamsHashOnClick = (schemeRegistrar: ISchemeRegistrar) => (): void =>
+  {
+    const { showNotification } = this.props;
+    copyToClipboard(schemeRegistrar.schemeToRegisterParamsHash);
+    showNotification(NotificationStatus.Success, "Copied to clipboard!");
```
Comment (orenyodfat) [inline src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx:58]: we do not have a generic function to copy to clipboard ?

File: src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx (hunk 1 above)
Comment (dkent600) [reply src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx:50]: sure.  frankly, the entire app should've been done that way.  I would refactor them all, but am respecting that we're trying to keep changes simple right now.

Comment (dkent600) [reply src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx:50]: done

File: src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx (hunk 2 above)
Comment (dkent600) [reply src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx:50]: no, I was lamenting that myself

Comment (orenyodfat) [reply src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx:50]: what about https://github.com/daostack/alchemy/blob/c13b354f84a8f86e22afc903527508ac8ff9b066/src/components/Notification/Notification.tsx#L38 ?

Comment (dkent600) [reply src/components/Proposal/ProposalSummary/ProposalSummarySchemeRegistrar.tsx:50]: that is just the notification, not the copy-to-clipboard itself.

**Pull request merged**